motr/setup.c

```diff
@@ -1426,6 +1436,8 @@ static int be_repair_zone_pcnt_get(struct m0_reqh *reqh,
 		       pver_obj->pv_u.subtree.pvs_attr.pa_K,
 		       pver_obj->pv_u.subtree.pvs_attr.pa_P,
 		       *repair_zone_pcnt);
+
+		M0_ASSERT(*repair_zone_pcnt <= M0_BC_REPAIR_ZONE_MAX_ALLOWED);
```

Comment:
Indentation needs to be corrected

Reply:
Hi Yeshpal, I don't see any indentation issue. Can you please check it once? 
Previous lines were arguments of a function call. Or please let me know If I missing something.